modules/administration/pages/channel-management.adoc

```diff
@@ -0,0 +1,37 @@
+[[channel-management]]
+= Channel Management
+// Antora entities
+ifndef::backend-pdf[]
+include::ROOT:partial$entities.adoc[]
+endif::[]
+// PDF entities
+ifdef::backend-pdf[]
+include::_partials/entities.adoc[]
+endif::[]
+
+Channels are a method of grouping software packages.
+
+In {productname}, channels are grouped into base and child channels, with base channels usually grouped by hardware architecture and software version, and child channels being related to a specific base channel.
```

Comment:
The base channel defines the OS (type, version and arch) while the child channels should be compatible to it.
When a system later gets assigned to a base channel, it is limited to only select the child channels of its base channel.

The relation base <=> child is all about compatibility. You should not use a SLES10 SP1 s390x base channel and define a Ubuntu universe 18.04 child channel below it.